recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
+  number: 0
+  script_env:
+    - MAKEFLAGS
```

Comment:
We are trying to figure out ways to eliminate boilerplate here. If you have thoughts on that point, please share in this issue ( https://github.com/conda-forge/staged-recipes/issues/535 ). Maybe we can come up with something that is mutually beneficial.